### PR TITLE
Update the website to match API Platform 2.2 features

### PR DESCRIPTION
**Pull request by dunglas**
ping @ginifizz (the new wording)

### CODE REVIEW

File: src/components/home/Ecosystem.js [hunk 1]
```diff
@@ -44,10 +44,10 @@ const Ecosystem = () => (
   <section className="home__part home__ecosystem">
     <div className="container">
       <h1 className="ecosystem__title">
-        The API Platform <strong>framework</strong>
+        The API Platform <strong>Framework</strong>
       </h1>
       <h5>
-        API Platform is a set of tools to help you building API-first projects
+        API Platform is a set of tools build and consume web APIs
```
Comment (raoulclais) [inline src/components/home/Ecosystem.js]: API Platform is a set of tools to build and consume web APIs

File: src/components/home/Ecosystem.js [hunk 2]
```diff
@@ -60,19 +60,19 @@ const Ecosystem = () => (
         <EcosystemCard
           image={schemaImage}
           link="/docs/schema-generator"
-          text="Instantly generates a PHP data model from the Schema.org vocabulary."
+          text="Instantly generates a PHP data model from the Schema.org vocabulary. Let the ORM creating the related tables."
```
Comment (raoulclais) [inline src/components/home/Ecosystem.js]: Let the ORM create the related tables

File: src/components/home/Giants.js [hunk 3]
```diff
@@ -3,30 +3,32 @@ import spider from 'images/giants.svg';
...
+          The development environment and the deployment mechanism leverage
+          <strong>Docker, Kubernetes and Helm</strong>.
         </p>
         <HomeList data={data} className="giants__list" />
         <p className="hidden-small">
-          However, API Platform is designed as a set of independent and reusable
-          components. You can perfectly use them in raw PHP projects or in
-          projects using other frameworks.
+          API Platform is also designed as a set of independent and reusable
+          components. You can perfectly use them in a standalone way, or integrate by them yourself
```
Comment (raoulclais) [inline src/components/home/Giants.js]: integrate them by yourself

File: src/components/home/Ecosystem.js (hunk 1 above)
Comment (soyuka) [reply src/components/home/Ecosystem.js]: missing word `to`

**Pull request merged**